benchmark/utils/CxxTests/GlobalVars.h

```diff
@@ -0,0 +1,21 @@
+#ifndef BENCHMARK_GLOBAL_VARS_H
+#define BENCHMARK_GLOBAL_VARS_H
+
+static int globalInt = 42;
+static constexpr int globalConstexprInt = 42;
+static float globalFloat = 42;
+static constexpr float globalConstexprFloat = 42;
+
+struct BigObject {
+  char buff[512];
+};
+static BigObject globalBigObject = BigObject();
+static constexpr BigObject globalConstexprBigObject = BigObject();
+
+struct HasStaticConstexprMembers {
```

Comment:
Funny enough, these tests actually fail because of undefined symbols without #35311.

Reply:
Planning to 1) land this 2) test #35311 3) add these tests to #35311 4) land #35311.